mchf-eclipse/drivers/audio/cw/cw_gen.c

```diff
@@ -226,7 +226,7 @@ static const float sm_table[CW_SMOOTH_TBL_SIZE] =
 
 void CwGen_SetSpeed()
 {
-    ps.dit_time         = 1650/ts.keyer_speed;      //100;
+    ps.dit_time         = 1920/ts.keyer_speed;      //100;
 }
```

Comment:
It is nearly impossible that your device runs that much faster. In this case things like PC USB connections would not work for instance. Did you measure a pause as well?